flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateChunkReaderTest.java

```diff
@@ -68,27 +81,28 @@ public void testBufferRecycledOnSuccess() throws IOException, InterruptedExcepti
                     .readChunk(stream, serializer.getHeaderLength(), handler, "channelInfo", 0);
         } finally {
             checkState(!handler.requestedBuffers.isEmpty());
```

Comment:
```suggestion
            assertThat(handler.requestedBuffers).isNotEmpty();
```
It seems `checkState` could be replaced with `assertThat` since the latest gives a better message